config/application.rb

```diff
@@ -9,7 +9,7 @@
 
 module Labwhere
   class Application < Rails::Application
-    config.load_defaults 7.2
+    config.load_defaults 8.1
```

Comment:
note: the only concern would be calling first, last etc when there is no primary key and no order specified.